File: examples/python/loadModel/sketch.py
from p5 import *

# global teapot


# def preload():
# global teapot
# teapot = loadModel('assets/teapot.obj', True)


def setup():
  global teapot
  # print(pInst)
  createCanvas(400, 400, WEBGL)
  # background('black')
  teapot = loadModel('assets/teapot.obj', True)


def draw():
  coordinateMode(TOP_LEFT)
  model(teapot)
  # background('black')
#   drawTickAxes()

#   fill('hotpink')
#   circle(200, 200, 50)

File: tests/js/star/sketch.js
function setup() {
  createCanvas(400, 400);
}

function draw() {
  coordinateMode(BOTTOM_LEFT);

  background("black");
  drawTickAxes();
  fill("hotpink");
  star(100, 120, 69);

  coordinateMode(TOP_LEFT);
  star(300, 120, 69);
}
